packages/alea-frontend/pages/_app.tsx

```diff
@@ -72,6 +72,26 @@ function CustomApp({ Component, pageProps }: AppProps) {
         clearInterval(interval);
       }
     }, 10);
+    const currentBuildId = process.env.NEXT_PUBLIC_BUILD_ID;
+    const pollBuildId = setInterval(async () => {
+      try {
+        const res = await fetch('/api/build-id');
+        const { buildId: latestBuildId } = await res.json();
+
+        if (currentBuildId && latestBuildId !== currentBuildId) {
+          console.log(`New build detected: ${currentBuildId} → ${latestBuildId}`);
```

Comment:
Add an alert here:
`alert('Refreshing to switching to a newer version of the application');